send/banned_facility_users

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+export SERVICE_NAME="banned_facility_users"
+
+python3 generic_send.py "$1" "$2" "$3"
```

Comment:
any particular reason python version of the send script is used instead of perl?

Reply:
i think there's continuous work in progress by @zlamalp on turning all send scripts using generic perl send to use generic python send. security issues i believe?